winui/Shimmer/Overview.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title: Overview of WinUI Shimmer control | Syncfusion
+description: Learn about the introduction of Syncfusion WinUI Shimmer Control (SfShimmer) with essential features and more.
+platform: WinUI
+control: Shimmer
+documentation: ug
+---
+
+# WinUI Shimmer Overview
+
+The WinUI Shimmer control enhances the responsiveness of your application by displaying a shimmer effect during content loading or background processing.
+
+![WinUI Shimmer control overview](Shimmer_images/Overview_images/winui_shimmer_overview.gif)
```

Comment:
@GilbertmanojSF3971 Change the overview image.

Reply:
@Manivannan-E Sir Changed the Overview image.